cmd/clusterctl/clusterdeployer/bootstrap/minikube/minikube.go

```diff
@@ -61,7 +63,7 @@ func WithOptionsAndKubeConfigPath(options []string, kubeconfigpath string) *Mini
 		}
 		return true
 	}() {
-		options = append(options, fmt.Sprintf("profile=%s", minikubeClusterName))
+		options = append(options, fmt.Sprintf("profile=%s", minikubeClusterNamePrefix+util.RandomString(5)))
```

Comment:
The options set here do not seem to be used by `Delete()`

Reply:
@detiber Thanks for your comment.